api/v1alpha1/api.go

```diff
@@ -437,6 +437,15 @@ const (
 	//
 	// https://learn.microsoft.com/en-us/azure/ai-services/openai/reference#api-specs
 	APISchemaAzureOpenAI APISchema = "AzureOpenAI"
+	// APISchemaGeminiVertexAI is the schema followed by Gemini models hosted on GCP's Vertex AI platform.
+	//
+	// https://cloud.google.com/vertex-ai/docs/reference/rest/v1/projects.locations.endpoints/generateContent?hl=en
```

Comment:
i think we should add the documentation about this schema that this requires the BackendSecurityPolicy to be configured and attached since this transformation will use project-id from there ?

Reply:
makes sense, can you point to where i need to add the documentation?